client/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:lts
+LABEL maintainer="Beatrice Williams"
+LABEL application="Task Manager Momo UI"
+
+COPY package.json package-lock.json /app/
+WORKDIR /app
+
+RUN npm install
+COPY . /app/
+
+ENV NODE_ENV production
+
+EXPOSE 3000
+
+CMD ["node_modules/.bin/next", "dev"]
```

Comment:
My research says you should not run commands as root, but use the "node" user. Doing so here caused file permission errors. Since this isn't a real production site, I am running root to save time.